src/core/runtime.hpp

```diff
@@ -237,8 +237,6 @@ inline bool stop(SHWire *wire, SHVar *result = nullptr) {
 
   SHLOG_TRACE("stopping wire: {}", wire->name);
 
-  wire->dispatcher.trigger(SHWire::OnStopEvent{wire});
```

Comment:
Might need to have two of those, one in the coroutines path the other in the inline wire call path

Reply:
see last comment